docs/siem/detections/api/exceptions/api-create-exception-item.asciidoc

```diff
@@ -0,0 +1,318 @@
+[[exceptions-api-create-exception-item]]
+=== Create exception item
+
+Creates an exception item and associates it with the specified
+<<exceptions-api-create-container, exception container>>.
+
+See <<lists-api-overview>> for information about creating exception items from
+lists, such as a list of IP addresses or host names.
+
+NOTE: Before creating exception items, you must create an exception container.
+
+[IMPORTANT]
+==============
+Endpoint rule exception items cannot use
+<<lists-api-overview, lists>> (the `list` element in the `entries` object), and
+the following fields cannot be used in exception queries (as `field` values in
+the `entries` object):
+
+* `file.Ext.quarantine_path`
+* `file.Ext.quarantine_result`
+* `process.entity_id`
+* `process.parent.entity_id`
+* `process.ancestry`
+==============
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/exception_lists/items`
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`comments` |comments[] a|Array of `comment` fields:
+
+* `comment` (string): Comments about the exception item.
+
+|No, defaults to empty array.
+
+|`description` |String |Describes the exception item. |Yes
+|`entries` |<<entries-object-schema, entries[]>> |Array containing the
+exception queries. Boolean `AND` logic is used to evaluate the relationship
+between array elements. If you want to use `OR` logic, create a separate
+exception item. |No, defaults to an empty `entries` array.
```

Comment:
We'll need to update this, this can no longer be an empty array or `undefined` as an exception item is useless without it.